app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -139,22 +148,19 @@ def open_claims
     all_claims['open_claims']
   end
 
-  def rated_disabilities
-    response = EVSS::DisabilityCompensationForm::Service.new(auth_headers).get_rated_disabilities
-    response.rated_disabilities
+  # fetch, memoize, and return all of the veteran's rated disabilities from EVSS
+  def all_rated_disabilities
```

Comment:
Renamed to emphasize that EVSS returns _all_ rated disabilities. This method now also memoizes the response, so that the updated serviced-connected logic can more conveniently use `all_rated_disabilities`.